Library/Homebrew/cmd/audit.rb

```diff
@@ -321,6 +321,11 @@ def audit_urls
       problem "\"http://ftpmirror.gnu.org\" is preferred for GNU software (url is #{u})."
     end
 
+    # Check if main url is used again as a mirror
+    if (urls & :mirrors).length
```

Comment:
`:mirrors` is a symbol, not an array. This line will raise a TypeError. Ignoring that, this condition would always be true, because `Array#length` always returns a value.